src/payment-providers/tsys/tsys.spec.ts

```diff
@@ -270,4 +278,54 @@ describe('tsys', () => {
           });
     });
   });
+
+  describe('perform live test', () => {
+    it('should successfully receive a token from TSYS', (done) => {
+      (<any> fetchMock)._unMock();
+      (<any> window).fetch('https://cortex-gateway-stage.cru.org/cortex/tsys/manifest')
+        .then((response: Response) => {
+          if (!response.ok) {
+            throw 'Error fetching deviceId and manifest from EP';
+          }
+          return response.json();
+        })
+        .then((tsysData: { deviceId: string, manifest: string }) => {
+          tsys.init('staging', tsysData.deviceId, tsysData.manifest);
+          tsys.encrypt('4111111111111111', '123', 12, new Date().getFullYear() + 1)
+            .subscribe(response => {
+                expect(response).toEqual({
+                  responseCode: 'A0000',
+                  status: 'PASS',
+                  message: 'Success',
+                  expirationDate: '12/2016',
+                  cvv2: '123',
+                  tsepToken: jasmine.stringMatching(/^[A-Za-z0-9]{16}$/),
+                  maskedCardNumber: '1111',
+                  cardType: 'V',
+                  transactionID: jasmine.stringMatching(/^\d{7,8}$/)
```

Comment:
I don't know where `tsepToken` or`transactionID` are documented either. If that `stringMatching` meets with current values, then I'm fine with leaving it for now.